src/python/espressomd/thermostat.pyx

```diff
@@ -17,13 +17,14 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
 from __future__ import print_function, absolute_import
+from decorator import decorator
```

Comment:
`decorator.decorator`'s job is pretty trivial. We shouldn't need to add a module and intoduce a dependency just to do that, you can probably do it in five lines yourself. [`decorator`](https://github.com/micheles/decorator/blob/master/src/decorator.py) uses `exec` inside, so it's a pretty terrible hack anyway!